tests/integration/basics.rs

```diff
@@ -0,0 +1,41 @@
+extern crate predicates;
+
+use super::helpers::*;
+
+use assert_cmd::prelude::*;
+use predicates::prelude::*;
+use std::process::Command;
+
+#[test]
+fn simple_cargo_thing() {
+    let template = dir("template")
+        .file(
```

Comment:
long run i'd love these as fixtures, inline files get messy quick (is fine for now, but just noting)